pjmedia/src/pjmedia/rtcp.c

```diff
@@ -384,17 +384,20 @@ PJ_DEF(void) pjmedia_rtcp_rx_rtp2(pjmedia_rtcp_session *sess,
         expected = last_seq - sess->seq_ctrl.base_seq;
 
         if (expected > sess->received)
-            count = expected - sess->received;
+            loss = expected - sess->received;
 
+        if (loss > sess->stat.rx.loss)
+            count = loss - sess->stat.rx.loss;
         period = count * sess->dec_pkt_size * 1000 / sess->clock_rate;
         period *= 1000;
```

Comment:
minor: move this period calculation to inside the `loss_period` update block below?